setup/bundles.rst

```diff
@@ -69,7 +69,8 @@ PHPUnit test report:
 
 .. code-block:: terminal
 
-    $ phpunit
+    # this command is available after running "composer require phpunit"
+    $ ./bin/phpunit
```

Comment:
This is a third-party bundle, so I think we should use the full component name, not the alias, right?

Reply:
The alias is for the bridge. So that looks good to me.